src/pipeline/DeviceNode.cpp

```
@@ -67,6 +68,19 @@ void DeviceNode::setLogLevel(dai::LogLevel level) {
     }
 }
 
+void DeviceNode::logTiming(const std::shared_ptr<spdlog::async_logger>& logger,
+                           std::chrono::steady_clock::time_point tAbsoluteBeginning,
+                           std::chrono::steady_clock::time_point tGotInput,
+                           std::chrono::steady_clock::time_point tProcessed,
+                           std::chrono::steady_clock::time_point tAbsoluteEnd) {
+    logger->trace("{} took {}ms, getting input {}ms, processing {}ms,  sending output {}ms",
+                  this->getName(),
+                  std::chrono::duration_cast<std::chrono::microseconds>(tAbsoluteEnd - tAbsoluteBeginning).count() / 1000,
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DEPTHAI_NLOHMANN_DEFINE_TYPE_INTRUSIVE is a macro then please configure it.

(unknownMacro)